content/TrueCommand/Administration/Settings/SAMLAD.md

```diff
@@ -0,0 +1,216 @@
+
+         ![EditedTcCertificate](/images/SAML/EditedTcCertificate.png "Edited Tc Certificate")
+      
+      6. Click **Save** (or <kbd>Ctrl-S</kbd>) and then close Notepad, and then Powershell.
+
+   d. Select **Browse** in the **Configure Certificate** Relying Party Trust Wizard window. 
+      The **Encryption Certificate** window opens.
+
+      ![WizardConfigureCertificateSelectBrowseCropped](/images/SAML/WizardConfigureCertificateSelectBrowseCropped.png "Wizard Configure Certificate Select Browse Cropped")
+   
+   e. Locate the <file>tc.cer</file> file (<file>C:/local disk/users/QE</file> and then select the <file>tc.cer</file>). 
+      Click **Open** to view the **Configure Certificate** window and see information about the certificate. 
+      Click **Next** to move on to the **Configure URL** window.
+
+4. Configure the URL. In the **Configure URL** window:
+
+   a. Select **Enable support for the SAML 2.0 WebSSO protocol**. 
+
+      ![WizardSelectEnableSupportForSAML2_0WebSSOProtocolCropped](/images/SAML/WizardSelectEnableSupportForSAML2_0WebSSOProtocolCropped.png "Wizard Select Enable Support For SAML 2.0 Web SSO Protocol Cropped")
+
+   b. Enter the TrueCommand login URL (http://*IP:PORT*/saml/acs) in the **Relying party SAML 2.0 SSO service URL** field. 
+      *IP:PORT* is your TrueCommand system IP and port address.
+
+   c. Click **Next** to continue to the **Configure Identifiers** window.
+
+5. Configure the SAML identifiers. 
+   
+   a. Type **truecommand-sml** into the **Relying party trust identifier** field and click **Add**.
+      
+      ![WizardConfigureIdentifiersTypeNameCropped](/images/SAML/WizardConfigureIdentifiersTypeNameCropped.png "Wizard Configure Identifiers Type Name Cropped")
+   
+      b. Click **Next** to move on to the **Choose Access Control Party** window. 
+    
+      c. Click **Next** again to view the **Ready to Add Trust** window, 
+         then click **Next** once more and then select **Close**. 
+       
+      ![WizardClickFinishCropped](/images/SAML/WizardClickFinishCropped.png "Wizard Click Finish")
+       
+
+6. Modify the newly-created Relying Party Trust.
+
+   a. Select the new SAML Relying Party Trust. 
+      With it highlighted, select **Properties** in the **Actions** menu on the right side of the screen to open the **Properties** window. 
+
+      ![SelectRelyingPartyTrustAndProperties](/images/SAML/SelectRelyingPartyTrustAndProperties.png "Select Relying Party Trust And Properties")
+   
+   b. Select the **Endpoints** tab, then click **Edit** to open the **Edit Endpoint** window.
+
+   c. Change the **Index** value to **1** and click **OK**. 
+      
+      ![EditPropertiesEndpointsChangeIndexValueCropped](/images/SAML/EditPropertiesEndpointsChangeIndexValueCropped.png "Edit Properties Endpoints Change Index Value Cropped")
+
+   d. Click the **Add SAML** button to open the **Add an Endpoint** window.
```

Comment:
My test directions:
Click **Add SAML**
Select Endpoint type **SAML Logout**.

Reply:
If this is changes you may need to change the screenshot for the 2nd endpoint "SAML Logout".

Reply:
I went back to the original instructions from JT's test script and the process we worked out and selecting Endpoint type **SAML Logout** that was not part of the instructions so this is new and probably why I missed it. I reviewed the original images in sequence and for this part there is no place to select the type **SAML Logout**